src/components/AvatarCropModal/Slider.tsx

```diff
@@ -40,18 +35,20 @@ function Slider(props) {
     // default behaviour of cursor - I-beam cursor on drag. See https://github.com/Expensify/App/issues/13688
     return (
         <View
-            ref={ControlSelection.blockElement}
+            ref={(el) => {
+                ControlSelection.blockElement<View>(el as SelectionElement<View>);
```

Comment:
```suggestion
                ControlSelection.blockElement(el as HTMLElement | null);
```

Based on my suggestion.

Reply:
Done here: https://github.com/Expensify/App/pull/34238/commits/e9a397960ca982ae30b5ca0d17442200e62f19b8